parsers/svg-to-jsx/svg-to-jsx.spec.ts

```diff
@@ -72,4 +72,21 @@ describe('Svg to jsx', () => {
     });
     return;
   });
+  it('Validate kebabCase on attribute', async () => {
+    const tokens = seeds().tokens.filter(({ name }) => name === 'activity');
+    const result = await svgToJsx(tokens as InputDataType, undefined, libs as LibsType);
+    if (result instanceof Error) return fail(result);
+    expect(result).toHaveLength(1);
+    expect(result[0].value.content).toMatch(/strokeWidth/)
+    expect(result[0].value.content).toMatch(/strokeLinecap/)
+    expect(result[0].value.content).toMatch(/strokeLinejoin/)
+    return;
+  });
+  it('Validate kebabCase on attribute with snapshot', async () => {
+    const tokens = seeds().tokens.filter(({ name }) => name === 'activity');
+    const result = await svgToJsx(tokens as InputDataType, undefined, libs as LibsType);
+    if (result instanceof Error) return fail(result);
+    expect(result[0].value.content).toMatchSnapshot();
+    return;
+  });
```

Comment:
What do you prefer?

Reply:
I prefer to avoid the usage of snapshots because it's too macro.